update-grml-rescueboot

```diff
@@ -19,8 +19,10 @@ for binary in "${!bin_to_pkg[@]}" ; do
   fi
 done
 
-if ! [ -r /usr/share/keyrings/debian-keyring.gpg ] ; then
-  echo "ERROR: File /usr/share/keyrings/debian-keyring.gpg not found." >&2
+keyringname="/usr/share/keyrings/debian-keyring"
+keyring="${keyringname}.gpg"
+if [ -z "${keyring}" ] ; then
+  echo "ERROR: File ${keyringname}.gpg not found." >&2
```

Comment:
This a bit redundant.  How about: `s/${keyringname}.gpg/${keyring}`

Reply:
the new error message has become: 
`"ERROR: File ${keyringname}.{pgp,gpg} not found."`